security/protecting-personal-or-identifying-information.md

```diff
@@ -0,0 +1,17 @@
+# Protecting Personal or Identifying Information
+
+Data privacy and security should be made a priority when developing software in
+an effort to protect personal or identifying information.
+
+Some individuals, due to their backgrounds or circumstances, might be at a
+higher risk of harmful consequences from privacy violations.
```

Comment:
```suggestion
Data privacy and security is important for everyone. It is especially vital for individuals who, due to their backgrounds or circumstances, might be at a higher risk of harmful consequences from privacy violations.
```

Rewording since we all benefit from PII protection